core/discovery/reducer/mocks.go

```diff
@@ -80,3 +80,11 @@ func conditionIsProvider1(provider market.ServiceProposal) bool {
 func conditionIsStreaming(provider market.ServiceProposal) bool {
 	return provider.ServiceType == "streaming"
 }
+
+func fieldID(proposal market.ServiceProposal) interface{} {
```

Comment:
Why do we returning `interface{}` here? Don't we have a predefined type for `ID`?